filesearch/filesearch.go

```diff
@@ -159,7 +160,10 @@ func FindConfigurationIncludes(configFile string, includes []string) ([]string,
 				for _, match := range matches {
 					addFile(match)
 				}
+			} else if err == nil {
+				clog.Tracef("no match: %s", include)
 			} else {
+				clog.Errorf("no match: %s ; %v", include, err)
```

Comment:
the error is returned and displayed after `cannot load configuration file`. So I think here we should add the missing information to the error returned instead of displaying it. Like:
```
return nil, fmt.Errorf("%w: %q", err, include)
```

Reply:
Makes sense

Reply:
Returns an error like "`syntax error in pattern: "/tmp/[--]"`" now.

Reply:
Perfect, thanks